src/root/Multiplexer/index.js

```diff
@@ -193,13 +205,16 @@ const mapStateToProps = (state) => ({
   userResponse: userResponseSelector(state),
   languageResponse: languageResponseSelector(state),
   currentLanguage: currentLanguageSelector(state),
+  disasterTypes: state.disasterTypes,
+  disasterTypesSelect: disasterTypesSelectSelector(state),
```

Comment:
@frozenhelium @batpad I added `disasterTypesSelect` here, but simply whenever I tried to get it with `state.disasterTypesSelect` it was empty so thus why I just used `disasterTypesSelect: disasterTypesSelectSelector(state),` in all places.

Reply:
That's weird, the end result for both should be the same

Reply:
I'll ticket this out too for later